locallib.php

```diff
@@ -2131,6 +2165,28 @@ function bigbluebuttonbn_process_meeting_events($bigbluebuttonbn, $jsonobj) {
         // Enqueue a task for processing the completion.
         bigbluebuttonbn_enqueue_completion_update($bigbluebuttonbn, $userid);
     }
+    // Process Overall Meeting Data.
+    $overrides = [
+        'recordid' => $recordid,
+        'meetingid' => $meetingid,
+        'userid' => null // Non user specific events are not linked to a user.
+    ];
+    $data = $jsonobj->{'data'};
```

Comment:
Can this be accessed with just ->data ? If im remembering the way the curly braces work correctly

Reply:
Yes they are functionally equivalent. The one time you might want to use the curly ones is when you are using a variable inside it.

I've only copied the style of the surrounding block @ line 2181, but happy to change them all to suit.